docs/index.rst

```diff
@@ -108,3 +119,4 @@ constructive feedback.
 .. _Code of conduct: https://ubuntu.com/community/code-of-conduct
 .. _Contribute: https://github.com/canonical/ubuntu-advantage-client/blob/main/CONTRIBUTING.md
 .. _IRC channel on Libera: https://kiwiirc.com/nextclient/irc.libera.chat/ubuntu-server
+.. _our Discourse forum: https://discourse.ubuntu.com/t/ubuntu-pro-beta-invitation/30957
```

Comment:
Alternatively, we could link to the "Ubuntu Pro" topic on discourse https://discourse.ubuntu.com/c/ubuntu-pro/116
Then people can create new posts or browse the existing posts.
What do you think?

Reply:
I did have a scan through all the posts on there, but it looks a bit...empty...so I went for the one that was best constructed and most helpful to new users.

That said, I can change it to the topic if you think that's better, or we could create a new post specifically to welcome peoples' feedback/questions/etc on the pro client.

Reply:
hmmmm Grant is right, this one is for the beta. Pointing to the topic makes more sense? I would expect it to start empty as people didn't have the time to fill it yet (:

Reply:
Should we start a "welcome" post, in that case?

Reply:
On reflection, I think Grant was right in the first place - I've updated the link as he suggested to the one for the Pro Discourse main channel.